Further Let's Encrypt ratelimit warnings (#9627)
traefik/traefik: docs/content/https/acme.md

You can configure Traefik to use an ACME provider (like Let's Encrypt) for automatic certificate generation.

!!! warning "Let's Encrypt and Rate Limiting"

>     Note that Let's Encrypt API has [rate limiting](https://letsencrypt.org/docs/rate-limits). These last up to __one week__, and can not be overrided.

Reviewer: ```suggestion
    Note that Let's Encrypt API has [rate limiting](https://letsencrypt.org/docs/rate-limits). These last up to __one week__, and can not be overridden.
    
    When running Traefik in a container this file should be persisted across restarts. 
    If Traefik requests new certificates each time it starts up, a crash-looping container can quickly reach Let's Encrypt's ratelimits.
    To configure where certificates are stored, please take a look at the [storage](#storage) configuration.
```

ACME certificates are stored in a JSON file that needs to have a `600` file mode.

> If you are using Traefik in a container make sure this file is preserved across restarts. If traefik requests new certificates each time it starts up, a crash-looping container can quickly reach Let's Encrypt's ratelimits. You may be left without valid certificates for a week.

Reviewer: ```suggestion
```

> If you are using Traefik in a container make sure this file is preserved across restarts. If traefik requests new certificates each time it starts up, a crash-looping container can quickly reach Let's Encrypt's ratelimits. You may be left without valid certificates for a week.

Reviewer: ```suggestion
```

> If you are using Traefik in a container make sure this file is preserved across restarts. If traefik requests new certificates each time it starts up, a crash-looping container can quickly reach Let's Encrypt's ratelimits. You may be left without valid certificates for a week.
> 

Reviewer 2: ```suggestion
```
